plugins/module_utils/user.py

```diff
@@ -141,6 +104,14 @@ def get_tls_requires(cursor, user, host):
         return requires or None
 
 
+def get_valid_privs(cursor):
+    cursor.execute("SHOW PRIVILEGES")
+    show_privs = [priv[0].upper() for priv in cursor.fetchall()]
+    other_privs = ['ALL', 'ALL PRIVILEGES', 'GRANT', 'REQUIRESSL']
```

Comment:
Maybe should we keep this as a constant at the top of the file, just in case we need to add more in the future (in place of the original VALID_PRIVS, something like EXTRA_PRIVS)

Reply:
+1

Reply:
Good point, done!